src/utilities/base-maps-dates.ts

```diff
@@ -0,0 +1,49 @@
+import { GoogleAuth, auth } from 'google-auth-library';
+import { GoogleSpreadsheet } from 'google-spreadsheet';
+
+const sheetId = '1XnncmhWrIjntlaMfQnMrlcCTyl9e2i-ztbvqryQYXDc';
+
+const scopes = ['https://www.googleapis.com/auth/spreadsheets', 'https://www.googleapis.com/auth/drive'];
+
+let client;
+if (import.meta.env.PROD) {
+  console.log('using ci credentials');
+  client = auth.fromJSON(JSON.parse(import.meta.env.GOOGLE_PRIVATE_KEY));
+  client.scopes = scopes;
+} else {
+  client = new GoogleAuth({
+    scopes,
+  });
```

Comment:
**question**: in dev, should we use fake data to avoid rate limiting?

Reply:
Since it's only one call to get the range of cells, I wasn't worried about it.